CalculatorApp/Calculator.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalculatorApp
+{
+    public class Calculator
+    {
+        public int add(string numbers)
```

Comment:
method name should start with capital, and be more descriptive
suggestion: 
**CalculateSumFromInput**